### PR TITLE
DENTAL UPDATE!

### PR DESCRIPTION
**Pull request by Crystalwarrior**
People now have teeth that can be knocked out of their mouths. Targeting head will have the chance to do that, targeting mouth improves the chances even more!"
Without teeth you will have a lisp. There's two new surgeries added: teeth reinsertion surgery and dental implant surgery. Both surgeries require you to target mouth with damp rag for them to activate. Teeth reinsertion surgery can be performed bare-handed and doesn't require any tools. You can remove teeth that way, too!
Dental implant surgery requires a drill and a pill. You will insert the pill into the target's mouth and the target will be able to activate the pill whenever they please!
If you lost your original teeth you can go up to an autolathe, open up MEDICAL category and make some replacement teeth. They can still have dental implants in them.
Also made collapsed rollerbeds fit in backpacks

### CODE REVIEW

**Comment by ghost** [comment]
approvequick m8 its good update

**Comment by AliiTea** [comment]
10/10 russian code

File: tgstation.dme [hunk 1]
```diff
@@ -6,6 +6,77 @@
...
+#define FILE_DIR "sound/misc"
+#define FILE_DIR "sound/piano"
+#define FILE_DIR "sound/spookoween"
+#define FILE_DIR "sound/violin"
+#define FILE_DIR "sound/voice"
+#define FILE_DIR "sound/voice/complionator"
+#define FILE_DIR "sound/voice/meeseeks"
+#define FILE_DIR "sound/vox"
+#define FILE_DIR "sound/vox/secret"
+#define FILE_DIR "sound/weapons"
```
Comment (nalarmg) [inline tgstation.dme]: None of this should be a thing.

Comment (Crystalwarrior) [inline tgstation.dme]: Huh, it was added automatically though.Do I remove all sound filedirs EXACTLY?

**Comment by lcass** [comment]
I think we should have a dentist maybe even an abandoned dentist area with scary tools somewhere in maint.

**Comment by ghost** [comment]
do you need drapes to perform surgery

**Comment by Chronitonity** [comment]
" Both surgeries require you to target mouth with damp rag for them to activate. Teeth reinsertion surgery"

Can you still use a sheet or drapes for this, because you probably should let players use those for consistency sake.

**Comment by Chronitonity** [comment]
Looks like you can use drapes unless im stupid, the faster this is merged the faster I can finish up my project.

**Comment by ghost** [comment]
says nothing about drapes in the code seems you dont need to do surgery? self surgery? WHO KNOWS

**Comment by Nebyab** [comment]
Yeah sure

**Comment by danhibiki** [comment]
Change from damp rag to drapes for consistency please.

**Comment by Crystalwarrior** [comment]
Why the fuck are you taking about damp rag?
This is using drapes like every surgery. This was merge-ready for 12 hours holy shit

EDIT: Just realised I wrote damp rag in changelog. Holy fucking shit.

**Pull request merged**